### ISSUE TITLE
Column labels forced to lowercase

### ISSUE THREAD

**Issue by sdknij** [awslabs/aws-data-wrangler#279]
**Describe the bug**
With version 1.4, column labels are turned into lowercase (e.g. "MY LABEL" becomes "my label").

**To Reproduce**
```
data = {'UPPER':['UPP 1', 'UPP 2', 'UPP 3'],
        'lower':['low 1', 'low 2', 'low 3'],
        'MiXeD':['MiX 1', 'mIx 2', 'Mix 3']}
df1 = pd.DataFrame(data)
print(f'Original dataframe:\n\n{df1}\n')
print(f'Headers for df1:{df1.columns}\n')

print(f'--- WRITING TO PARQUET ---\n')
wr.s3.to_parquet(df1, myPath + '/TeSt')
df2 = wr.s3.read_parquet(inputPath + '/TeSt')
print(f'Dataframe read from parquet:\n{df2}')
print(f'\n* But what about df1?\n')
print(f'Original dataframe:\n{df1}\n')
print(f'Headers for df1:{df1.columns}\n')

Original dataframe:

   UPPER  lower  MiXeD
0  UPP 1  low 1  MiX 1
1  UPP 2  low 2  mIx 2
2  UPP 3  low 3  Mix 3

Headers for df1:Index(['UPPER', 'lower', 'MiXeD'], dtype='object')

--- WRITING TO PARQUET ---

Dataframe read from parquet:
   upper  lower mi_xe_d
0  UPP 1  low 1   MiX 1
1  UPP 2  low 2   mIx 2
2  UPP 3  low 3   Mix 3

* But what about df1?

Original dataframe:
   upper  lower mi_xe_d
0  UPP 1  low 1   MiX 1
1  UPP 2  low 2   mIx 2
2  UPP 3  low 3   Mix 3

Headers for df1:Index(['upper', 'lower', 'mi_xe_d'], dtype='object')

```

**Comment by igorborgest** [comment]
Thanks @sdknij !

I've just fixed it in our development branch, and will be released soon on version `1.5.0`.

Now, we have the argument `sanitize_columns`.

Could you test it, and give us some feedback?

To install the development branch:

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by igorborgest** [comment]
Released on version [1.5.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.5.0)!
